style.css

```diff
@@ -561,6 +580,26 @@ textarea {
     width: 70%;
   }
 
+  /* work section */
+  .container {
+    display: flex;
+    flex-direction: row;
+    margin: 30px 142px;
+  }
```

Comment:
- [ ] The images in the project cards should be alternated like the first project should have the project picture on the **left**, the one under it on the **right**, and then the one under it on the **left**, and so on, like the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848), kindly change it to match that. 😁

![image](https://user-images.githubusercontent.com/20567503/178043875-50095356-8bd0-4fc0-93ae-e7932cccce22.png)